documentation/content/configuration/pat.md

```diff
@@ -5,6 +5,8 @@ title: Github Personal Access Token
 
 The token generated below will only have read-only access to public information.
 
+Why does HACS need this? The GitHub API lets HACS do 60 calls an hour without an access token, with a token HACS gets 5000 calls an hour.
```

Comment:
This is a good starting point, but it needs more information so it does not confuse more than it answers.
Preferably with links to the github API, and the usage limits.

I think @bokub has a plan of how this should be.